practice-02/js/index.js

```diff
@@ -1,12 +1,25 @@
 import myJson from '../database/products.json' assert {type: 'json'};
 
+/**
+ * This function is used to filter products by name and save them on localStorage
```

Comment:
Why we need to save them on localStorage

Reply:
Yes, local Storage is unnecessary, I removed localStorage